pulsar-proxy/src/test/java/org/apache/pulsar/proxy/server/ProxyWithJwtAuthorizationTest.java

```diff
@@ -297,6 +299,26 @@ public void testUpdatePartitionNumAndReconnect() throws Exception {
         Assert.assertEquals(messageSet, receivedMessageSet);
         consumer.close();
         producer.close();
+
+        // revoke produce/consume permission
+        admin.topics().revokePermissions(topicName, CLIENT_ROLE);
```

Comment:
This test doesn't cover your changes:

1. Increasing this topic partition
2. Revoking this topic permission

Reply:
Good catch.

Reply:
I guess it has cover

1. Line274 increasing partition from 2 to 4
2. Line304 revoke topic permission

Reply:
Oh, I see.